### DISCUSSION TITLE
Force code reviews for all commits by specific team and/or user in orgianization

### DISCUSSION THREAD

**Discussion by pixiemediaweb** [community/community#28298] [Other Feature Feedback, Questions, & Ideas]
Hi,

Can someone help me out with this please.   In our organisation we have jnr's and snrs.  I've created sub teams and associated the users.  What I'd like to do now is say: for anyone in the jnr team, any commit they make in any organisation repo should force a code review.

Is this possible?

**Comment by rsclarke** [comment]
Regardless of whether they are senior or junior, it is often good practice for members to conduct code review.  To ensure this is done, create a branch protection rule with;

* **Require a pull request before merging**
* **Require approvals**
* **Required number of approvals before merging**, select as appropriate
* **Dismiss stale pull request approvals when new commits are pushed**
* **Require review from Code Owners**

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

The `CODEOWNERS` file (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) in the repository will list the user(s) or team(s) who may approve the code review.  In your scenario this could be the senior team.
